### ISSUE TITLE
Unique color mappings

### ISSUE THREAD

**Issue by patgarcia** [sourceoptics/source_optics#167]
I think it would be useful to have unique [color mappings](https://github.com/altair-viz/altair/issues/921#issuecomment-395416682) for the author's field. 

While reviewing the [files view](https://drive.google.com/open?id=1ZiFul36J8UIMH4SMDF7IdXWY48hasK9K), I was scanning for a specific author's name and felt a little cognitively burdened by having to look for the color of that author on each chart.

This is in the neighborhood of #130

**Comment by mpdehaan** [comment]
Yeah file view is about 48 hours old, and I had the same thought as soon as I saw it.

This django package looks interesting:

https://stackoverflow.com/questions/39859224/how-to-use-html5-color-picker-in-django-admin

We could also add this to the repo, for the repo graph comparison feature.
